README.md

```diff
@@ -29,4 +29,13 @@ Now add a dependency for the library:
 </dependency>
 ```
 
-Start by reading up on [`Validator`](http://download.eclipse.org/lyo/docs/validation/2.3.0.M3/org/eclipse/lyo/validation/Validator.html) Javadoc.
+Start by reading the library's main interface class [`Validator`](http://download.eclipse.org/lyo/docs/validation/2.3.0.M3/org/eclipse/lyo/validation/Validator.html) on Javadoc.
+
+You can also review the library's [`unit test cases`](https://github.com/eclipse/lyo-validation/tree/master/src/test/java/org/eclipse/lyo/validation) that shows how to use this interface. 
```

Comment:
backticks are for `source code`, no need to use them for normal link text

Reply:
will fix.